app/views/components/widget/_modal.html.erb

```diff
@@ -12,17 +12,23 @@
 <% else %>
   <div
     class="<%= form.prefix "usa-modal__content" %> fba-modal-dialog">
-    <div class="<%= form.prefix "usa-modal__main" %>">
-      <%= render "components/widget/no_modal", form: form do %>
-        <%= render partial: 'components/forms/footer', locals: { form: form } %>
-      <% end %>
+    <div>
+      <div class="<%= form.prefix "usa-modal__main" %> padding-bottom-0 padding-top-0">
+        <%= render "components/widget/no_modal", form: form %>
+      </div>
+      <%= render partial: 'components/forms/footer', locals: { form: form } %>
     </div>
     <%- unless form.delivery_method == "inline" %>
-      <button class="<%= form.prefix "usa-modal__close" %> fba-modal-close"
+      <button class="<%= form.prefix "usa-modal__close" %> usa-button fba-modal-close"
```

Comment:
would this be `<%= form.prefix "usa-button" %>` ?

Reply:
Only the modal classes are prefixed.